app/public/test/jquery.clipurl.spec.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
```

Comment:
@subotic I was able to add QUnit tests for the new plugin. However, these are bot automatically run.

Reply:
Maybe we could find a way so that tests can be run directly in the browser and in NodeJS: https://github.com/PabloVallejo/try-qunit.

Reply:
Ok, I found a way to run the tests from the console with `npm t` (https://github.com/dasch-swiss/dsp-tangoh-app/pull/17/commits/f7641ba618a0ffca93f82e2808799d87ddf615c8)